src/Auth.ts

```diff
@@ -306,6 +309,10 @@ export class Auth {
         logger.logToStderr('No existing refresh token. Starting new device code flow...');
       }
 
+      if (Cli.getInstance().getSettingWithDefaultValue<boolean>(settingsNames.autoOpenBrowserOnLogin, false)) {
+        this.open('https://aka.ms/devicelogin');
```

Comment:
Rather than using a hardcoded URL, let's use the URL that comes back from AAD. That way, we're not affected if the URL changes and also can properly support different clouds that each have their own URL.

Reply:
Oh right!

btw I was thinking about enhancing (in another issue to submit) if we could extend the `login` command with extra option, to let the user choose if he wants to use the paperclip to copy the device code when selected, so that he would just have to paste it in the browser once opened. What do you think?

Reply:
Love it! Would you mind creating a new issue?

Reply:
Done #2321 😀